setup.py

```diff
@@ -262,11 +262,16 @@ def run_setup(extensions):
 if "--no-cython" not in sys.argv:
     try:
         from Cython.Build import cythonize
-        cython_candidates = ['cluster', 'concurrent', 'connection', 'cqltypes', 'marshal', 'metadata', 'pool', 'protocol', 'query', 'util']
-        compile_args = [] if is_windows else ['-Wno-unused-function']
-        extensions.extend(cythonize(
-            [Extension('cassandra.%s' % m, ['cassandra/%s.py' % m], extra_compile_args=compile_args) for m in cython_candidates],
-            exclude_failures=True))
+        # cython_candidates = ['cluster', 'concurrent', 'connection', 'cqltypes', 'metadata',
```

Comment:
Are you suggesting that we should not cythonize the core files? My trials showed 3-10% improvement for write-intensive workloads (where SerDes does not dominate).